src/Hl7.Fhir.Core/Serialization/ElementNavFhirReader.cs

```diff
@@ -123,8 +120,31 @@ private bool verifyXmlSpecificDetails(IElementNavigator child)
                 throw Error.Format($"Xml node of type '{xmlDetails.NodeType}' is unexpected at this point", this);
         }
 
+        public bool MoveToNext(string nameFilter = null)
```

Comment:
Since the methods below are one-liners and you also have methods like LineNumber using the new C# syntax, you should also write those three below with the new syntax. Consistency is key :)

Eg:
`public bool MoveToNext(string nameFilter = null) => _current.MoveToNext(nameFilter);`

Reply:
will do

Reply:
Funny, all these methods were generated by a Visual Studio refactoring (implement interface by forwarding to a private member variable).  It will generate lambdas for properties, but obviously not for methods!